MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -95,6 +95,11 @@
     For iOS, this will be ignored if MSALWebviewTypeSystemDefault is chosen. */
 @property (nullable) WKWebView *customWebview;
 
+#if TARGET_OS_IPHONE
+@property (nullable, weak, nonatomic) UIViewController *parentViewController;
```

Comment:
@unpluggedk, can you make sure to add those into configuration object?

Reply:
#534